lib/templates/plugin/plugin.php.tpl

```diff
@@ -437,15 +437,15 @@ class {{class_name}}Plugin extends Plugin
     /**
      * Handler for the {{events.event}} event
      *
-     * @param EventObject $event The event to process
+     * @param EventInterface $event The event to process
```

Comment:
This works, but I think it would be better to use the same approach as the one used in the Order plugin.

```
<?php
use Blesta\Core\Util\Events\Common\EventInterface;

...

public function {{events.callback}}(EventInterface $event) { }
```

Reply:
Updated